ocs/defaults.py

```diff
@@ -48,3 +50,11 @@
 ROOK_API_VERSION = 'ceph.rook.io/v1'
 OCP_API_VERSION = 'project.openshift.io/v1'
 OPENSHIFT_REST_CLIENT_API_VERSION = 'v1'
+TEMPLATES_DIR = "templates"
+TEMP_YAML = os.path.join(TEMPLATES_DIR, "temp.yaml")
+
+CEPHFILESYSTEM = load_yaml_to_dict(
+    os.path.join(
+        "templates/ocs-deployment", "cephfilesystem_new.yaml"
```

Comment:
Depends on https://github.com/red-hat-storage/ocs-ci/pull/111/files for template file location, we can modify this after that is merged.